extensions/amp-gfycat/validator-amp-gfycat.protoascii

```diff
@@ -36,6 +36,10 @@ tags: {  # <amp-gfycat>
     mandatory: true
   }
   attrs: {
+    name: "autoplay"
```

Comment:
minor nit: we like to keep attributes in alphabetical order. if not too much trouble, move above `data-gfyid`